checker/src/visitors.rs

```diff
@@ -886,6 +890,13 @@ impl<'a, 'b: 'a, 'tcx: 'b, E> MirVisitor<'a, 'b, 'tcx, E> {
         from_hir_call: bool,
     ) {
         debug!("default visit_call(func: {:?}, args: {:?}, destination: {:?}, cleanup: {:?}, from_hir_call: {:?})", func, args, destination, cleanup, from_hir_call);
+        // This offset is used to distinguish any local variables that leak out from the called function
+        // from local variables of the callee function.
+        // This situation arises when a structured value stored in a local variable is assigned to
+        // a field reachable from a mutable parameter.
+        // We assume that no program that does not make MIRAI run out of memory will have more than
```

Comment:
Reminds me of this change to Python 3.7:

    More than 255 arguments can now be passed to a function, and a function can now have more than 255 parameters.

Reply:
We all assume something or other. 1 million is more than 640K, so sure it is going to be enough. :-)